_data/banking.yml

```diff
@@ -84,6 +84,14 @@ websites:
       - hardware
     doc: https://www.avanza.se/kundservice.html/1335/tvafaktorsinloggning/d264248e-bea3-48a0-bf2d-157309734649
 
+  - name: Baden-Württembergische Bank
```

Comment:
Is the correct name `Baden-Württembergische Bank` or `BW Bank`?

Reply:
Their legal info (https://www.bw-bank.de/de/home/toolbar/impressum.html) says:
> Die Baden-Württembergische Bank (BW-Bank) ist ...
The long name is a bit of a mouthful, so they shortened it to BW-Bank. However, I would leave it in so that users searching for the long name will still find it. Maybe even "Baden-Württembergische Bank (BW-Bank)"?

Reply:
That works :+1:

Reply:
Since the search apparently also looks at the URL, we don't even have to include "BW-Bank" here. I think the original commit should be good to go then.